app/controllers/state_file/questions/ny_w2_controller.rb

```diff
@@ -9,14 +9,21 @@ def self.show?(intake)
       end
 
       def index
-        @w2s_with_metadata = @w2s.map do |w2|
-          dfw2 = w2.state_file_intake.direct_file_data.w2s[w2.w2_index]
-          {
-            w2: w2,
-            employer_name: dfw2.EmployerName,
-            wages_amount: dfw2.WagesAmt,
-          }
+        if @w2s.length == 1
+          redirect_to action: :edit, id: @w2s[0].w2_index
```

Comment:
If there is only one W2, we don't perform any validation?

Reply:
If there is only 1 W2, we bypass the index page and take users directly into the edit for that W2